pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: http2
-version: 0.1.4
+version: 0.1.5
```

Comment:
This can/should be 0.1.4+1

No new features – just a fix

Reply:
I've always thought the `+N`'s were silly. It's build metadata, which should be ignored when determining precedence. Yes, I know we pinned ourselves to 2.0.0-rc1 of semver to get around this, since the final 2.0.0 release specifically says to ignore build metadata, but why?

Semver also says that the patch version should increase for backwards compatible bugfixes, which is what this is.

(And you could rightfully argue that the 0.1.4 release should have been 0.2.0, since I added new, backwards compatible, public API...)